packages/cf-component-heading/src/HeadingCaptionTheme.js

```diff
@@ -0,0 +1,7 @@
+export default () => ({
+  color: '#434148',
```

Comment:
https://github.com/cloudflare/cf-ui/blob/master/packages/cf-style-const/src/variables.js

I agree it should be in `cf-style-const`